src/Language/Rust.ts

```diff
@@ -266,6 +283,15 @@ class RustRenderer extends ConvenienceRenderer {
         }
     }
 
+    private visibility(): string {

```

Comment:
Great work! if you put `get` before the method name, this becomes a property and you can refer to it as `this.visibility`.

Reply:
`get_visibility`, `getVisibility`, or `getvisibility`?

Reply:
`get visibility()`